src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const prop of actions) {
+    if (prop.type === 'clear') {
```

Comment:
try use switch case when you want to check a value for multiple possible results